R/FilterState.R

```diff
@@ -254,13 +301,34 @@ FilterState <- R6::R6Class( # nolint
             tags$span(tags$strong(private$get_varname())),
             tags$span(private$get_varlabel(), class = "filter-card-varlabel")
           ),
-          if (isFALSE(private$is_locked())) {
-            actionLink(
-              inputId = ns("remove"),
-              label = icon("circle-xmark", lib = "font-awesome"),
-              class = "filter-card-remove"
-            )
-          },
+          div(
```

Comment:
I think it should be a dropdown menu with these buttons instead of listing two icons, what do you think @chlebowa @lcd2yyz ?

There is also CSS issue with long label which goes under these icons. CSS rule is made in a way that label is broken into another line when it reaches remove button.

![image](https://github.com/insightsengineering/teal.slice/assets/6959016/382e988f-9610-4218-a099-a813d777fa08)

Reply:
> I think it should be a dropdown menu with these buttons instead of listing two icons, what do you think @chlebowa @lcd2yyz ?

A dropdown menu means two clicks per action so if one wants to go back three steps, that would be six clicks.

> There is also CSS issue with long label which goes under these icons. CSS rule is made in a way that label is broken into another line when it reaches remove button.
> 
> ![image](https://user-images.githubusercontent.com/6959016/252280366-382e988f-9610-4218-a099-a813d777fa08.png)

Should we reopen [this](https://github.com/insightsengineering/teal.slice/issues/336) then?

Reply:
I like the two icons, it's easy access and elegant, provided that we can 1) fix CSS of the variable label, 2) add tooltip on hover with delay/expire to show prompt like "Undo" and "Reset".